pkgs/shells/zsh/nix-zsh-completions/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "nix-zsh-completions";
-  version = "unstable-2023-01-30";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = "nix-zsh-completions";
-    rev = "6a1bfc024481bdba568f2ced65e02f3a359a7692";
-    hash = "sha256-aXetjkl5nPuYHHyuX59ywXF+4Xg+PUCV6Y2u+g18gEk=";
+    rev = version;
```

Comment:
```suggestion
    rev = "refs/tags/${versions}";
```

Reply:
Why would you change it like that, it currently builds fine o.O

Reply:
It is a recommended practice that maintainers told me to follow from now on.
In most cases it is not required but in some situations it disambiguates tags from branches.

Reply:
Could you send a example I haven't seen that before :eyes:

Reply:
I don't have a specific example but here is the comment since when I am doing it systematically: https://github.com/NixOS/nixpkgs/pull/218429#discussion_r1118096472